SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialStaticComponentView.cpp

```diff
@@ -70,9 +69,6 @@ void USpatialStaticComponentView::OnAddComponent(const Worker_AddComponentOp& Op
 	case SpatialConstants::SPAWN_DATA_COMPONENT_ID:
 		Data = MakeUnique<SpatialGDK::SpawnData>(Op.data);
 		break;
-	case SpatialConstants::SINGLETON_COMPONENT_ID:
-		Data = MakeUnique<SpatialGDK::Singleton>(Op.data);
-		break;
```

Comment:
We have 0 use for this component within workers, it's just used to make singletons always relevant, saw no need to store this here